### ISSUE TITLE
Excel copies formulas from other rows even when the formula has been set to empty string, null or undefined for that row.

### ISSUE THREAD

**Issue by harri-rajala-koivu-solutions** [OfficeDev/office-js#1916]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

When adding column data to the table, Excel copies formulas from other rows even when the formula has been set to empty string, null or undefined for that row.

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: PC desktop
* Host [Excel, Word, PowerPoint, etc.]: Excel
* Office version number: Version 2105 (Build 14026.20246)
* Operating System: Windows Version 2004 (Os Build 19041.985)
 
## Expected behavior
Excel should not copy formulas from other rows when the formula has been set to empty string, null or undefined for that row.
It should work without any workaround (= add extra empty row). 
See the example code for more information. 

## Current behavior
Excel copies formulas from other rows even when the formula has been set to empty string, null or undefined for that row.
We found one workaround. Add an extra empty row to the table (by adding [undefined] to the first column) and then remove that extra row. 

See the example code for more information.

## Steps to reproduce
```typescript
await Excel.run(async (context: Excel.RequestContext) => {
    const sheet = context.workbook.worksheets.getActiveWorksheet();
    const createTable = async (tableName: string, range: string) => {
        // Delete existing table
        let table = sheet.tables.getItemOrNullObject(tableName);
        await context.sync();
        if (table && !table.isNullObject) {
            table.delete();
        }
        // Create table
        table = sheet.tables.add(range, true);
        table.name = tableName;
        table.getHeaderRowRange().values = [["Text", "Formula", "Number"]];
        return table;
    }
    sheet.getRange("A1:A3").values = [
        ["NOT OK"],
        ["Formulas should be only in the first and last row."],
        ["But for some reason formulas are also copied to 2-4 rows where the formula has been set to empty string, null and undefined."]
    ];
    const tableNotOk = await createTable("TableNotOk", "A4:C4");
    tableNotOk.getHeaderRowRange().getCell(0, 0).getRowsBelow(5).values = [["a"], ["bb"], ["ccc"], ["dddd"], ["eeeee"]];
    tableNotOk.getHeaderRowRange().getCell(0, 1).getRowsBelow(5).values = [["=LEN(A5)"], [""], [null], [undefined], ["=LEN(A9)"]];
    tableNotOk.getHeaderRowRange().getCell(0, 2).getRowsBelow(5).values = [[11], [22], [33], [44], [55]];
    await context.sync();

    sheet.getRange("A12:A18").values = [
        ["OK"],
        ["To solve this issue, we found the following workaround."],
        ["We must add [undefined] to the end of the first column. So there will be 6 rows in this table."],
        ["Add data to all columns. So first columns has 5 data rows + one [undefined]. The second and last columns have 5 data rows."],
        ["So now we have 6 rows in this table (5 data rows and one empty row at the end of the table)."],
        ["But because there are only 5 data rows, we must delete the last row."],
        ["Now we have 5 rows in this table and formulas are only in the first and last row."]
    ];
    const tableOk = await createTable("TableOk", "A19:C19");
    tableOk.getHeaderRowRange().getCell(0, 0).getRowsBelow(6).values = [["a"], ["bb"], ["ccc"], ["dddd"], ["eeeee"], [undefined]];
    tableOk.getHeaderRowRange().getCell(0, 1).getRowsBelow(5).values = [["=LEN(A20)"], [""], [null], [undefined], ["=LEN(A24)"]];
    tableOk.getHeaderRowRange().getCell(0, 2).getRowsBelow(5).values = [[111], [222], [333], [444], [555]];
    tableOk.getDataBodyRange().getLastRow().delete(Excel.DeleteShiftDirection.up);
    await context.sync();
});

```

**Comment by jipinshi** [comment]
We can repro the issue, internal 5182488 was reported to track it.

**Comment by yuc014** [comment]
This issue is fixed. Please let us know if you are still experiencing issues.

**Comment by gingerjia** [comment]
@harri-rajala-koivu-solutions, sorry to let you know the fix for this issue has caused a perf regression and we need to turn off the fix  to mitigate the issue. Please use the workaround to unblock your side work first. 
And when we have any progress for the fix, will update here to let you know. Sorry for the inconvenience and thanks very much for your understanding!
